<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayBooks</class>
 <widget class="QDialog" name="DisplayBooks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1257</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="booksDataSet">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>1241</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="titleLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>681</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>379</y>
     <width>851</width>
     <height>131</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>849</width>
      <height>129</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="imgLbl">
   <property name="geometry">
    <rect>
     <x>879</x>
     <y>380</y>
     <width>361</width>
     <height>401</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="desc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>841</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="genre">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>851</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
